cache-config/testing/ort-tests/t3c_mode_test.go

```diff
@@ -58,50 +97,50 @@ func TestT3cBadassAndSyncDs(t *testing.T) {
 		// TODO: remove this when running ATS is added to the test framework
 
 		if err := os.Rename(`/opt/trafficserver/bin/traffic_ctl`, `/opt/trafficserver/bin/traffic_ctl.real`); err != nil {
-			t.Fatal("temporarily moving traffic_ctl: " + err.Error())
+			t.Fatalf("temporarily moving traffic_ctl: %v", err)
 		}
 
 		fi, err := os.OpenFile(`/opt/trafficserver/bin/traffic_ctl`, os.O_RDWR|os.O_CREATE, 755)
 		if err != nil {
-			t.Fatal("creating temp no-op traffic_ctl file: " + err.Error())
+			t.Fatalf("creating temp no-op traffic_ctl file: %v", err)
 		}
 		if _, err := fi.WriteString(`#!/usr/bin/env bash` + "\n"); err != nil {
 			fi.Close()
-			t.Fatal("writing temp no-op traffic_ctl file: " + err.Error())
+			t.Fatalf("writing temp no-op traffic_ctl file: %v", err)
 		}
 		fi.Close()
 
 		defer func() {
 			if err := os.Rename(`/opt/trafficserver/bin/traffic_ctl.real`, `/opt/trafficserver/bin/traffic_ctl`); err != nil {
-				t.Fatal("moving real traffic_ctl back: " + err.Error())
+				t.Fatalf("moving real traffic_ctl back: %v", err)
 			}
 		}()
 
 		// run badass and check config files.
-		if err := runApply("atlanta-edge-03", "badass"); err != nil {
-			t.Fatalf("ERROR: t3c badass failed: %v\n", err)
+		if err := runApply(cacheHostName, "badass"); err != nil {
+			t.Fatalf("t3c badass failed: %v", err)
 		}
 
 		// Use this for uid/gid file check
-		atsUser, err := user.Lookup(TrafficServerOwner)
+		atsUser, err := user.Lookup(trafficServerOwner)
 		var atsUid string
 		var atsGid string
 
 		if err != nil {
-			t.Logf("Unable to look up user: %s: %v", TrafficServerOwner, err)
+			t.Logf("Unable to look up user: %s: %v", trafficServerOwner, err)
 		} else {
 			atsUid = atsUser.Uid
 			atsGid = atsUser.Gid
 		}
 
 		for _, v := range testFiles {
```

Comment:
Just a comment: I wouldn't ask for more work in this PR, but IMO this loop would be a great case for `t.Run` subtests. Every iteration of the loop is its own test, that make sense independently and don't require unique setup.

Reply:
hey, sounds like a good idea to me